hasjob/templates/campaign_action_edit_header.html.jinja2

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="img-container col-xs-12 col-sm-3 col-sm-push-9 post-company-logo">
+    {% if campaign.banner_image %}<img alt="Logo" src="{{ campaign.banner_image }}"/>{% endif %}
+  </div>
+  <div class="col-xs-12 col-sm-9 col-sm-pull-3">
+    <h1><a href="{{ campaign.url_for() }}">{{ campaign.title }}</a></h1>
+    <p class="post-date">
+      {%- if campaign.state.is_live %}<strong>Live</strong>{% else %}{{ campaign.state.label }}{% endif %} —
```

Comment:
`LIVE` state also has a label and we can use that too. And here you're using `state.label` as label is defined as unicode string. And I've mostly defined labels as namedtuple and use `title` in templates etc. Is that fine? Or should we use a single convention?

Reply:
This construct was necessary to add the `<strong>` tag. I haven’t found a way to automate this.